webrtc.html

```diff
@@ -1525,6 +1525,10 @@
                         the corresponding media description.</p>
                       </li>
                       <li>
+                        <p>If <var>transceiver</var>'s <a>[[\Stopped]]</a> slot
+                        is <code>true</code>, abort these sub steps.</p>
```

Comment:
Is the meaning of "abort these sub steps" clear enough? Maybe it's time to break the giant list of "set a description" steps into different algorithms. Also, this is where `[[CurrentDirection]]` is set for a local description, but it's also set for a remote description further down.